sdk/identity/Azure.Identity/src/InteractiveBrowserCredentialOptions.cs

```diff
@@ -36,6 +37,12 @@ public class InteractiveBrowserCredentialOptions : TokenCredentialOptions, IToke
         /// </summary>
         public bool AllowUnencryptedCache { get; set; }
 
+        /// <summary>
+        /// Uri where the STS will call back the application with the security token. This parameter is not required if the caller is not using a custom <see cref="ClientId"/>. In
+        /// the case that the caller is using their own <see cref="ClientId"/> the value must match the redirect url specified when creating the application registration.
+        /// </summary>
+        public Uri RedirectUri { get; set; }
```

Comment:
Should it be `null` or `DefaultRedirectUrl` as default value?

Reply:
I could see an argument for both sides. I made it null to avoid the allocation but I could make a static readonly to accomplish the same. Currently there are other values which have a default when not set that we don't expose to the user such as `ClientId` so using null is a consistent with that as well.

Reply:
Ok